CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 - Unreleased
+  - Add `PyUntypedArray` as an untyped base type for `PyArray` which can be used to inspect arguments before more targeted downcasts. ([#369](https://github.com/PyO3/rust-numpy/pull/369))
```

Comment:
Let's mention that some methods (e.g., `len`) are moved to `PyUntypedArray`. I don't think it affects user codes in most cases, but it's theoretically a breaking change.

Reply:
Note that this is definitely a breaking, i.e. this will only be released as 0.19.0.